replicator.go

```diff
@@ -0,0 +1,304 @@
+package simplex
```

Comment:
// Copyright (C) 2019-2025, Ava Labs, Inc. All rights reserved.
// See the file LICENSE for licensing terms.